### PR TITLE
[helm-chart] Added global.PodLabels

### PR DESCRIPTION
**Pull request by balazs92117**
#### What this PR does
This PR adds global.podLabels which can add POD labels to PODs directly controlled by this chart (mimir services, nginx).

#### Which issue(s) this PR fixes or relates to
It's a feature, not a bugfix.

#### Checklist

- [ ] Tests updated
- [X] Documentation added
- [X] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 * [CHANGE] Compactor: the deprecated configuration parameter `-compactor.consistency-delay` has been removed. #5050
 * [CHANGE] Store-gateway: the deprecated configuration parameter `-blocks-storage.bucket-store.consistency-delay` has been removed. #5050
 * [FEATURE] Query-frontend: add `-query-frontend.log-query-request-headers` to enable logging of request headers in query logs. #5030
+* [ENHANCEMENT] Add global.podLabels which can add POD labels to PODs directly controlled by this chart (mimir services, nginx).
```
Comment (dimitarvdimitrov) [inline CHANGELOG.md:22]:
```suggestion
* [ENHANCEMENT] Add global.podLabels which can add POD labels to PODs directly controlled by this chart (mimir services, nginx). #5055
```

**Review by dimitarvdimitrov — APPROVED** [review]
Thanks for the contribution!

**Comment by flxbk** [comment]
The CHANGELOG has just been cut to prepare for the next Mimir release. Please rebase `main` and eventually move the CHANGELOG entry added / updated in this PR to the top of the CHANGELOG document. Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,7 @@
 * [CHANGE] The configuration parameters `-querier.iterators` and `-query.batch-iterators` have been deprecated and will be removed in Mimir 2.11. Mimir runs by default with `-querier.batch-iterators=true`, and starting from version 2.11 it will not be possible to change this. #5114
 * [CHANGE] Compactor: change default of `-compactor.first-level-compaction-wait-period` to 25m. #5128
 * [FEATURE] Query-frontend: add `-query-frontend.log-query-request-headers` to enable logging of request headers in query logs. #5030
+* [ENHANCEMENT] Add global.podLabels which can add POD labels to PODs directly controlled by this chart (mimir services, nginx). #5055
```
Comment (dimitarvdimitrov) [inline CHANGELOG.md:27]:
```suggestion
* [ENHANCEMENT] Add global.podLabels which can add POD labels to PODs directly controlled by this chart (mimir services, nginx). #5055
```

**Comment by dimitarvdimitrov** [comment]
Sorry for the delay and thank you for keeping this up to date @balazs92117! I pushed one commit to move the changelog entry to the helm-specific changelog file.

This will be included in the next helm release coming in the next couple of weeks (4.5.0)

**Pull request merged**